pages/manual/manual-gettingstarted-build.md

```diff
@@ -46,6 +46,16 @@ If you use a single-configuration generator, you can control the build mode usin
  Unless you specifically need to debug the application, we recommend setting the build mode to `RelWithDebInfo` for good performance, while still getting reasonable stacktraces for debugging and error reporting.
 " %}
 
+### Recommended setup (Optional)
+
+1. Build Type: Set the build mode to `RelWithDebInfo` ([See this guide for Visual Studio](https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019))
+
+2. Build with multiple cores:
+    - Visual Studio: `Tools->Options->Projects and Solutions->Build and Run->maximum number of parallel project builds` We recommend 2-4 on an 8-core machine
```

Comment:
I have never set that one, might prefix it with something like if the machine is unusable while building one might set that one and "IVW_MULTIPROCESSOR_COUNT". I generally never set that either though... Think that also VS2022 is supposed to have better built in load balancing. But I don't remember where I read that...